internal/pyxis/builder_test.go

```diff
@@ -143,7 +143,7 @@ var _ = Describe("Pyxis Builder tests", func() {
 				)
 				Expect(err).ToNot(HaveOccurred())
 				Expect(input.TestResults.ID).To(Equal(results.ID))
-				Expect(input.TestResults.UserResponse.Image).To(Equal(results.UserResponse.Image))
+				Expect(input.TestResults.Image).To(Equal(results.Image))
```

Comment:
Was this change as a result or side effect of a linter suggestion ? or possibly a fix unrelated to the pr ?

Reply:
Ah i see, I was just curious. Should be good then 👍